python/tests/db/test_db.py

```diff
@@ -10,25 +10,26 @@
 import pytest
 import sys
 
-from mspasspy.ccore.seismic import Seismogram, TimeSeries
-from mspasspy.ccore.utility import dmatrix, ErrorSeverity, MetadataDefinitions, Metadata
+from mspasspy.ccore.seismic import Seismogram, TimeSeries, TimeSeriesEnsemble, SeismogramEnsemble
+from mspasspy.ccore.utility import dmatrix, ErrorSeverity, Metadata
+from pyspark import SparkConf, SparkContext
 
 from mspasspy.db.schema import MetadataSchema
 from mspasspy.util import logging_helper
 from bson.objectid import ObjectId
 from datetime import datetime
 
 sys.path.append("python/tests")
-sys.path.append("python/mspasspy/db/")
-from database import Database
-from client import Client
+from mspasspy.db.database import Database, read_distributed_data
+from mspasspy.db.client import Client
 from helper import (get_live_seismogram,
                     get_live_timeseries,
                     get_live_timeseries_ensemble,
                     get_live_seismogram_ensemble)
 
+
 class TestClient():
```

Comment:
The tests for Client should be in the test_client.py file. We should follow the rule of one test per module.

Reply:
done